internal/pkg/config/bgp_configs.go

```diff
@@ -2212,7 +2212,7 @@ type RouteReflectorState struct {
 }
 
 // struct for container bgp:config.
-// Configuraton parameters relating to route reflection
+// Configuration parameters relating to route reflection
 // for the BGP neighbor or group.
```

Comment:
this file is automatically generated from OpenConfig yang files and can't be modified directly. The first two typos come from the original yang files so I guess that we had better to leave them alone.

The last one comes from:
https://github.com/osrg/gobgp/blob/master/tools/pyang_plugins/gobgp.yang